lib/thinking_sphinx/real_time/populator.rb

```diff
@@ -12,9 +12,9 @@ def populate(&block)
 
     remove_files
 
-    scope.find_each do |instance|
-      transcriber.copy instance
-      instrument 'populated', :instance => instance
+    scope.find_in_batches do |instances|
+      transcriber.copy instances
+      instrument 'populated', :instance => instances
```

Comment:
This is for the console output when running ts:generate - I've updated things so this works again in 730269ea2b15ba0a19341c381ccc424e8c3198d9.